### ISSUE TITLE
Analytics Discovery - [10-10CG Form | Mulesoft Implementation and Response Time

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#35542]
## Context and Conclusions
What trend/measurable changes have you observed with your products?

- We'd like to be able to measure performance in terms of when a user hits "submit" on the 10-10cg form until they are returned a response (either success or fail).
- Currently we don't report on this metric regularly (I did find an old ticket that noted the response time was between 30 sec - 2 min. [Ticket 20320](https://app.zenhub.com/workspaces/vsa---caregiver-5fff0cfd1462b6000e320fc7/issues/department-of-veterans-affairs/va.gov-team/20320) )
  
  
## Acceptance Criteria
- [x] Documentation (e.g. product outline) updated: [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/health-benefits/healthcare-application/product-outline.md)
- [ ] Follow-on implementation/discovery tickets created 

## How to configure this issue
- Labeled with team-name

**Comment by bsmartin-ep** [comment]
@SKasimow -

I built a quick query to calculate the AVG difference between `caregivers-10-10cg--submission` events and `caregivers-10-10cg--(submission-successful|submission-failed|submission-failed-client)` events and for 2022-01-17 I see the following milliseconds...

```
11141.409448818902 ms
```

...which seems very high to me. I think I need to trace a specific session to see if anything unusual is happening.  Are the events firing at the correct time?  Is there some sort of delay - like a pageview - before we get a response?

You said you expected this to be less than 5 seconds, correct?

**Comment by SKasimow** [comment]
@bsmartin-ep I haven't been able to find any documentation/performance history. The only things I've found is a ticket that mentions that it can take from 30 sec - 2 min for a response. I've opened a request to have one of our devs check Grafana logs to see what they show [Ticket 25536](https://app.zenhub.com/workspaces/vsa---10-10-team-5fff0cfd1462b6000e320fc7/issues/department-of-veterans-affairs/va.gov-team/35536).

Are you able to do a trace - that would be super helpful!
